conf/COROSYNC-MIB.txt

```diff
@@ -174,14 +182,15 @@ corosyncNoticesAppStatus NOTIFICATION-TYPE
        name and the new state (either 'connected' or 'disconnected')."
 ::= { corosyncNotices 3 }
 
-corosyncNoticesRRPStatus NOTIFICATION-TYPE
+corosyncNoticesLinkStatus NOTIFICATION-TYPE
```

Comment:
Hi, Honza, in fact, corosyncNoticesLinkStatus already has 5 filelds in my patch. Is there anything else missing?